airbyte-integrations/connectors/destination-mssql/src/main/java/io/airbyte/integrations/destination/mssql/MSSQLDestination.java

```diff
@@ -65,22 +65,29 @@ public JsonNode toJdbcConfig(JsonNode config) {
 
     List<String> additionalParameters = new ArrayList<>();
 
-    final StringBuilder jdbcUrl = new StringBuilder(String.format("jdbc:sqlserver://%s:%s;user=%s;password=%s;?",
-        // final StringBuilder jdbcUrl = new
-        // StringBuilder(String.format("jdbc:sqlserver://%s:%s;databaseName=%s;user=%s;password=%s;?",
+    final StringBuilder jdbcUrl = new StringBuilder(String.format("jdbc:sqlserver://%s:%s;;databaseName=%s;user=%s;password=%s;?",
         config.get("host").asText(),
         config.get("port").asText(),
-        // config.get("database").asText(),
+        config.get("database").asText(),
         config.get("username").asText(),
         config.get("password").asText()));
 
     if (config.has("ssl") && config.get("ssl").asBoolean()) {
-      additionalParameters.add("ssl=true");
-      additionalParameters.add("sslmode=require");
+      additionalParameters.add("encrypt=true");
+      if (config.has("trustServerCertificate") && config.get("trustServerCertificate").asBoolean()) {
```

Comment:
where would these params come from? I don't see this in `spec.json`

Reply:
This has been added to `spec.json`